peachjam/js/components/ProvisionEnrichments/ProvisionEnrichment.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <la-gutter-item
+    :anchor.prop="anchorElement"
+  >
+    <div class="card">
+      <div class="card-body">
+        <p>
+          {{ $t('This provision was declared unconstitutional') }}
+          <span v-if="enrichment.judgment">{{ $t('by') }} <a :href="enrichment.judgment.frbr_uri">{{ enrichment.judgment.title }}</a></span>
+          <span v-if="enrichment.date_deemed_unconstitutional">{{ " " }}{{ $t('on') }} {{ enrichment.date_deemed_unconstitutional }}</span>
+          <span>.</span>
```

Comment:
remove all of this, it's too much.

Reply:
those badges look odd, we can change to use a p tag